src/Parser/Core/Modules/StatTracker.js

```diff
@@ -236,45 +247,61 @@ class StatTracker extends Analyzer {
       case SPECS.RESTORATION_DRUID:
         return 66667;
       default:
-        throw new Error('Mastery hasn\'t been implemented for this spec yet.');
+        console.error('Mastery hasn\'t been implemented for this spec yet.');
+        return 99999999;
```

Comment:
Ok, so instead of crashing the StatDisplay will now simply display a wrong answer?

Reply:
It will show 0, which is obviously wrong

Reply:
Won't it show base mastery, which is less obviously wrong? Also, do you want to push to display something that is obviously wrong?

I get that you're probably looking at including all of the Mastery conversion factors as not your problem, but I would argue that now that you're adding stat percentage display it *is* your problem. 

The good news is there is apparently a simpler formula for the different specs Mastery conversion, which I am looking into.

Reply:
I looked into it briefly, it did not seem like an easy to tackle problem and I figured if it's obviously wrong it won't be misleading. It should be easy to implement for spec maintainers once they get to it.

Reply:
https://i.imgur.com/G1F1ECN.jpg